<ui version="4.0" >
 <class>QCAAxisBase</class>
 <widget class="QWidget" name="QCAAxisBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="layoutDirection" >
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="axisFrame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="axisNameLabel" >
        <property name="text" >
         <string>Energy</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>16</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCALCDNumber" name="axisReadback" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>80</width>
          <height>28</height>
         </size>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="smallDecimalPoint" >
         <bool>false</bool>
        </property>
        <property name="numDigits" >
         <number>10</number>
        </property>
        <property name="segmentStyle" >
         <enum>QLCDNumber::Flat</enum>
        </property>
        <property name="value" stdset="0" >
         <double>10.896</double>
        </property>
        <property name="blankOnInvalid" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="axisUnitsLabel" >
        <property name="text" >
         <string>eV</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>16</width>
          <height>28</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCACounter" name="axisCounter" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="layoutDirection" >
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="numButtons" >
         <number>1</number>
        </property>
        <property name="hasToolTip" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCACounter</class>
   <extends>QwtCounter</extends>
   <header>qcacounter.h</header>
  </customwidget>
  <customwidget>
   <class>QCALCDNumber</class>
   <extends>QLCDNumber</extends>
   <header>qcalcdnumber.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
